src/Kmd.Logic.Gateway.Automation/Client/Extensions/GatewayClient.ProductExtensions.cs

```diff
@@ -663,26 +664,7 @@ public async Task<HttpOperationResponse<object>> CustomUpdateProductWithHttpMess
                 try
                 {
                     result.Body = SafeJsonConvert.DeserializeObject<IDictionary<string, IList<string>>>(responseContent, this.DeserializationSettings);
-                }
-                catch (JsonException ex)
-                {
-                    httpRequest.Dispose();
-                    if (httpResponse != null)
-                    {
-                        httpResponse.Dispose();
-                    }
-
-                    throw new SerializationException("Unable to deserialize the response.", responseContent, ex);
-                }
-            }
-
-            // Deserialize Response
-            if ((int)statusCode == 422)
-            {
-                responseContent = await httpResponse.Content.ReadAsStringAsync().ConfigureAwait(false);
-                try
-                {
-                    result.Body = SafeJsonConvert.DeserializeObject<IDictionary<string, IList<string>>>(responseContent, this.DeserializationSettings);
+                    Console.WriteLine(result.Body);
```

Comment:
Same here - should we print that to console?

Reply:
We are displaying the appropriate error message. If we are not printing this, then the error message is not displayed in console

Reply:
Removed Console.writeline